Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "rusqlite"
-version = "0.11.0"
-authors = ["John Gallagher <jgallagher@bignerdranch.com>"]
+version = "0.11.1"
+authors = ["John Gallagher <jgallagher@bignerdranch.com>, Mason Chou <masonchou95035@gmail.com>"]
 description = "Ergonomic wrapper for SQLite"
-repository = "https://github.com/jgallagher/rusqlite"
+repository = "https://github.com/Mason-Chou/rusqlite"
```

Comment:
Yes this is my bad, I cloned the repository to do some edits/changes. Forgot to change it back.